src/pages/help-centre/deposits-and-withdrawals.tsx

```diff
@@ -182,9 +182,19 @@ const WithdrawMaestroMastercard = ({ text }: ArticleProps) => (
     <ArticleWrapper>
         <StyledHeader as="h4">{text}</StyledHeader>
         <Text>
-            {localize(
-                'Mastercard and Maestro card withdrawals are only available for UK clients. If you are not from the UK, you’ll need to use a withdrawal method that’s available in your country.',
-            )}
+            <Localize
+                translate_text="Making withdrawals using Maestro and Mastercard is not possible in your country. You can check the <0>list of  our payment methods</0> to find a withdrawal method available to you."
+                components={[
+                    <ExternalLink
+                        to={'/payment-methods/'}
+                        target="_blank"
+                        external
+                        weight="bold"
+                        rel="noopener noreferrer"
+                        key={0}
+                    />,
```

Comment:
`payment-methods` is not an external link

Reply:
<ExternalLink /> is just styled component, not a localized function